### PR TITLE
feat: add infrastructure components (traefik, prometheus, grafana) add one more broker and coordinator

### PR DESCRIPTION
**Pull request by Mala1180**

### CODE REVIEW

**Review by kelvin-olaiya — COMMENTED** [review]
A few questions:
- Are the ./k8s files autogenerated?
- What's the difference between the `media-server-service` & the `media-server-deployment`?

File: common/src/infrastructure/events/KafkaConsumer.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default class KafkaConsumer {
         topics: topics,
         fromBeginning: fromBeginning
       })
-
+      console.log('Consumer subscribed to topics: ', topics)
```
[the remark this answers is not in the document]
Comment (Mala1180) [reply common/src/infrastructure/events/KafkaConsumer.ts:27]: I need that for now, i swear i'll remove it.

File: kafka/docker-compose.yml [hunk 2]
```diff
@@ -1,39 +1,74 @@
...
+    container_name: revue-zookeeper-2
+    ports:
+      - ${ZOOKEEPER_PORT_2}:2181
+    environment:
+      - ALLOW_ANONYMOUS_LOGIN=yes
+    networks:
+      - my-net
+
+  revue-kafka-1:
     image: bitnami/kafka:latest
```
[the remark this answers is not in the document]
Comment (Mala1180) [reply kafka/docker-compose.yml:24]: It is for distributed stuffs (coordination, consensus, etc...), key-value based

**Comment by Mala1180** [comment]
> A few questions:
> 
> * Are the ./k8s files autogenerated?
> * What's the difference between the `media-server-service` & the `media-server-deployment`?

- yes, it waa untracked but the i devided to track them because there is no automation that rebuild that files... at the moment
- services and deployments are differents kubernetes abstractions, the first handles the access to the service, the second handles the lifecycle of pods (and containers)

**Review by kelvin-olaiya — DISMISSED** [review]

**Review by kelvin-olaiya — APPROVED** [review]

**Pull request merged**